core/app/models/spree/variant.rb

```diff
@@ -28,6 +28,10 @@ class Variant < Spree::Base
     attr_writer :rebuild_vat_prices
     include Spree::DefaultPrice
 
+    # Consider that not all platforms digest structured data in the same way,
+    # you might have to modify the output on the frontend or in feeds accordingly.
+    enum :condition, { damaged: "Damaged", new: "New", refurbished: "Refurbished", used: "Used" }, prefix: true
```

Comment:
Can we make use of I18n here please?

Reply:
Ah, get it. Thanks

Reply:
@rahulsingh321 we spoke about this. 

@tvdeyen the most practical way would be to return the value capitalized

Reply:
@rahulsingh321 please store all values as followed and return them the same way: 

1. `New` instead of `new`
2. `Refurbished` instead of `refurbished`
3. `Used` instead of `used`
4. `Damaged` instead of `damaged`

Reply:
The Solidus API in our opinion does not need to conform to another actor's exact API. If you need to interface with Google or other third-party services, either convert the return values of the API to uppercase in custom code that consumes the Solidus API or implement a custom API endpoint that returns the fields exactly as Google recommends.